.github/workflows/binary_artifact.yml

```diff
@@ -132,47 +132,40 @@ jobs:
            rm -rf .git/
            rm -rf thirdparty/tcc/.git/
            rm -rf vc/
-           rm -rf v_old.exe
+           rm -rf v_old
```

Comment:
I am not sure about that, have you tested in a VM or actual windows, in msys2, that the resulting file is not `v_old.exe` ?

Reply:
Good point. Was going to here being in a rush. Sorry.